.github/workflows/component-ci.yml

```diff
@@ -72,20 +72,21 @@ jobs:
           username: _json_key
           password: ${{ secrets.GOOGLE_CREDENTIALS }}
 
-      - name: Build and export to Docker (to GCP in master/main)
+      - name: Build and export to Docker (to GCP in master)
         uses: docker/build-push-action@v3
         with:
-          load: ${{ !(github.ref == 'refs/heads/master' || github.ref == 'refs/heads/main') }}
-          push: ${{ github.ref == 'refs/heads/master' || github.ref == 'refs/heads/main' }}
+          load: ${{ !(github.ref == 'refs/heads/master') }}
+          push: ${{ github.ref == 'refs/heads/master' }}
           cache-from: |
             ${{ env.IMAGEBASE }}:master
-            ${{ env.IMAGEBASE }}:main
           tags: |
             ${{ env.IMAGEBASE }}:${{ env.imagetag }}
             ${{ env.IMAGEBASE }}:latest
           labels: |
             ${{ inputs.component_name }}
           build-args: |
+            POSTURE=${{ github.ref == 'refs/heads/master' && 'operational' || 'dev' }}
```

Comment:
There is no ternary operator support in github actions but we have this workaround